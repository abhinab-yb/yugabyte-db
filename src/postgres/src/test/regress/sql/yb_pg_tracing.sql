SELECT is_yb_pg_tracing_enabled((SELECT pg_backend_pid()), NULL);

SELECT yb_pg_enable_tracing(pid => -1, query_id => NULL);

SELECT is_yb_pg_tracing_enabled(-1, NULL);

SELECT yb_pg_enable_tracing(query_id => NULL, pid => (SELECT pg_backend_pid()));

SELECT yb_pg_enable_tracing(-100, query_id => NULL);

SELECT yb_pg_enable_tracing(0, NULL);

SELECT yb_pg_disable_tracing(-1, NULL);

SELECT is_yb_pg_tracing_enabled((SELECT pg_backend_pid()), NULL);