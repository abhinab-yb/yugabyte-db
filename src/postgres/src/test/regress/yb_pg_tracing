# src/test/regress/yb_pg_tracing
#
####################################################################################################
# This testsuite contains testcases pertinent to YB TRACE
####################################################################################################
test: yb_pg_tracing
