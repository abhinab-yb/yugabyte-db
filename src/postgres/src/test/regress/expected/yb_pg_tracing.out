SELECT is_yb_pg_tracing_enabled((SELECT pg_backend_pid()), NULL);
 is_yb_pg_tracing_enabled 
--------------------------
 f
(1 row)

SELECT yb_pg_enable_tracing(pid => -1, query_id => NULL);
 yb_pg_enable_tracing 
----------------------
 t
(1 row)

SELECT is_yb_pg_tracing_enabled(-1, NULL);
 is_yb_pg_tracing_enabled 
--------------------------
 t
(1 row)

SELECT yb_pg_enable_tracing(query_id => NULL, pid => (SELECT pg_backend_pid()));
 yb_pg_enable_tracing 
----------------------
 t
(1 row)

SELECT yb_pg_enable_tracing(-100, query_id => NULL);
ERROR:  pid cannot be less than -1

SELECT yb_pg_enable_tracing(0, NULL);
 yb_pg_enable_tracing 
----------------------
 t
(1 row)

SELECT yb_pg_enable_tracing(query_id => NULL, -1);
ERROR:  positional argument cannot follow named argument
LINE 1: select yb_pg_enable_tracing(query_id => NULL, -1);
                                                      ^

SELECT yb_pg_disable_tracing(pid => -1, query_id => NULL);
 yb_pg_disable_tracing 
-----------------------
 t
(1 row)

SELECT is_yb_pg_tracing_enabled((SELECT pg_backend_pid()), NULL);
 is_yb_pg_tracing_enabled 
--------------------------
 f
(1 row)

